README.md

```
@@ -1 +1,163 @@
+- **Batch operations**: Apply changes to multiple files at once
+
+## Prerequisites
+
+- Visual Studio Code 1.74.0 or higher
+- Node.js 16.x or higher
+- npm 8.x or higher
+
+## Setup
+
+1. Clone the repository:
+   ```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)